x/twap/api_test.go

```diff
@@ -52,21 +62,27 @@ var (
 		baseTime.Add(20*time.Second),
 		sdk.NewDec(2),               // spot price 0
 		OneSec.MulInt64(10*10+5*10), // accum A
-		OneSec.MulInt64(3))          // accum B
+		OneSec.MulInt64(3),          // accum B
+		sdk.ZeroDec(),               // TODO: choose correct
+	)
 
 	tPlus20sp2ThreeAssetRecordAB, tPlus20sp2ThreeAssetRecordAC, tPlus20sp2ThreeAssetRecordBC = newThreeAssetPoolTwapRecordWithDefaults(
 		baseTime.Add(20*time.Second),
 		sdk.NewDec(2),                // spot price 0
 		OneSec.MulInt64(10*10+5*10),  // accum A
 		OneSec.MulInt64(3),           // accum B
-		OneSec.MulInt64(20*10+10*10)) // accum C
+		OneSec.MulInt64(20*10+10*10), // accum C
+		sdk.ZeroDec(),                // TODO: choose correct
+		sdk.ZeroDec(),                // TODO: choose correct
+		sdk.ZeroDec(),                // TODO: choose correct
+	)
```

Comment:
I need to add more tests where these values would be used.

Currently, this is set to zero to avoid existing arithmetic twap tests from failing due to geometric accumulators not being initialized.